src/mod.ts

```diff
@@ -5,7 +5,13 @@ import { ParsePayload } from "./parse-payload.ts";
 import { LoadFunctionModule } from "./load-function-module.ts";
 import { RunFunction } from "./run-function.ts";
 
-await (async () => {
+export const run = async function (functionDir: string) {
```

Comment:
Would it be fair to default `functionDir` to `functions/`?

Reply:
Since this particular code path would be used only by the Lambda context, I lean towards the existing no-default-and-fail-noisily-fast approach and instead put the onus of providing the right directory argument on the consumer (that is, on @curtisallen).

Reply:
Doesn't this PR [remove the existing default](https://github.com/slackapi/deno-slack-runtime/pull/9/files#diff-5990be9abcb61bed7171d8f186ffed0e189aef1fa9e1ac018ebc2f1830feb647L18) though? I just thought moving it from `LoadFunctionModule` to the `mod.ts` file used by lambda made sense here. Regardless, I think if @curtisallen approves we can move forward with the no default!